third_party/2and3/redis/client.pyi

```diff
@@ -81,6 +81,28 @@ class Redis(object):
     def pubsub(self, shard_hint: Any = ..., ignore_subscribe_messages: bool = ...) -> PubSub: ...
     def execute_command(self, *args, **options): ...
     def parse_response(self, connection, command_name, **options): ...
+    def acl_cat(self, category: Optional[Text]) -> List[str]: ...
+    def acl_deluser(self, username: Text) -> int: ...
+    def acl_genpass(self) -> Text: ...
+    def acl_getuser(self, username: Text) -> Union[None, Dict[str, Union[List[str], bool]]]: ...
```

Comment:
In cases like this, where the return value contains an `Union` (in this case `Union` inside the `Dict`, not the outer `Union`), we prefer to use `Any` instead. Otherwise the caller has to use `isinstance` checks each time they access the dict.

An even better alternative here would be to use a `TypedDict`, which can define exactly which fields and types the returned dict has. I'll leave the decision which of the three options to use (as is, `Any`, or a `TypedDict`) to you.

Also a style nit: Instead of `Union[None, ...]` we usually just use `Optional[...]`.

Reply:
Thanks for the detailed explanation; I chose to use `Any` - I would prefer to have used a `TypedDict`, but that requires me to name the type, which I would prefer not to do as I'm not the author of the library. (Unless there's a way to create an anonymous `TypedDict`  that I have not been able to find!)

Reply:
That's fine. What we usually do in these cases is creating a name marked with an underscore. There is also the `@typing.type_check_only` decorator, which hasn't gained traction, but we should use it more.